### ISSUE TITLE
Priority field in .swap unit is ignored

### ISSUE THREAD

**Issue by hexchain** [systemd/systemd#14524]
**systemd version the issue has been seen with**
> v244.1

**Used distribution**
 > Arch Linux

**Expected behaviour you didn't see**
 > Swap activated by systemd has the priority value specified in the swap unit file.

**Unexpected behaviour you saw**
 > Swap activated by systemd has a default priority value (-1, -2, -3, ...).

**Steps to reproduce the problem**
1. Create a new swap file and do `mkswap` on it, e.g. `/swapfile`
2. Put a swap unit file to `/run/systemd/system/swapfile.swap`:
  ```ini
  [Unit]
  Description=Swap file

  [Swap]
  What=/swapfile
  Priority=6000
  ```
3. `systemctl daemon-reload && systemctl start swapfile.swap`
4. Use `swapon` to observe that the priority is a negative value, not 6000.

It seems that in `src/core/swap.c`, `s->parameters_fragment.priority_set` is never `true` even though `Priority=` is specified in the unit file.

**Comment by poettering** [comment]
This was broken in 7477451b691d288dad67b4c8ce9e519e9b75770d. @topimiettinen

**Comment by poettering** [comment]
Fix waiting in #14533
